lib/tools/searchWeb/webDeepResearch.ts

```diff
@@ -1,5 +1,5 @@
-import getSearchWebTool from "./getSearchWebTool";
```

Comment:
YAGNI principle
- actual: no longer using the `getSearchWebTool` but the file remains in our codebase.
- required: delete the unused `getSearchWebTool` file. 🚮